.circleci/config.yml

```
@@ -20,3 +20,4 @@ jobs:
 
       - run: npm test
       - run: npm run lint
+      - run: npm run sonar
```

this can be sonar-yay
```suggestion
      - run: npm run sonar-yay
```